doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -419,11 +419,13 @@ and ``coqtop``, unless stated otherwise:
   *string* must be :n:`"@setting_name"`.
   See the :ref:`note above <interleave-command-line>` regarding the order
   of command-line options.
-:-compat *version*: Load a file that sets a few options to maintain
-  partial backward-compatibility with a previous version.  This is
-  equivalent to :cmd:`Require Import` `Stdlib.Compat.CoqXXX` with `XXX`
-  one of the last three released versions (including the current
-  version).  Note that the :ref:`explanations above
+:-compat *version*: same as ``-compat-from Stdlib Coq<version>``
```

Comment:
How is it going to work with the renaming to Rocq? Ideally, compat files should still be names `Coq819`, `Coq820` and `Rocq90`, no?

Reply:
My idea was simply to rename all files to Rocq* even if Rocq 8.18 has never been really a thing. Otherwise we can use you idea. Anyway since the command line option is `-compat X.Y` the precise name of the file doesn't play much of a role.

Reply:
I'd prefer to do the Coq -> Rocq renaming in a further small PR that I'll open just after this one, in order to avoid mixing unrelated changes (thr current PR has nothing to do with the renaming itself).

Reply:
> Anyway since the command line option is `-compat X.Y` the precise name of the file doesn't play much of a role.

Ah, that's a good point (less overengineered than my solution). We may still want to keep `Coq81X` modules just in case some people where directly depending on them though.

But then why not do this renaming of compat files to `RocqXY` in this PR as well?

Reply:
> I'd prefer to do the Coq -> Rocq renaming in a further small PR that I'll open just after this one, in order to avoid mixing unrelated changes (thr current PR has nothing to do with the renaming itself).

OK, but keep in mind that the second PR will have to update the changelog of this PR.